parm/config/config.fit2obs

```diff
@@ -0,0 +1,30 @@
+#! /usr/bin/env bash
+
+########## config.fit2obs ##########
+# Fit to Observations
+
+echo "BEGIN: config.fit2obs"
+
+# Get task specific resources
+. $EXPDIR/config.resources fit2obs
+
+export fit_ver="wflow.1.0"
+export fitdir="${BASE_GIT}/Fit2Obs/${fit_ver}"
+
+export HOMEcfs=${fitdir}
+export EXECcfs=${HOMEcfs}/exec
+export USHcfs=${HOMEcfs}/ush
```

Comment:
Are these even used?

Reply:
They are used within the Fit2Obs package.

Reply:
ok. These may need a new definition as fit2obs conforms to EE2.  But for a later PR.

Reply:
for now, I can define these in the fit2obs modulefile so they don't need to appear here.